Bundle/TwigBundle/DependencyInjection/VictoireTwigExtension.php

```diff
@@ -28,5 +28,8 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter(
             'victoire_twig.responsive', $config['responsive']
         );
+
+        $twigConfig['globals']['victoire_twig_responsive'] = $container->getParameter('victoire_twig.responsive');
```

Comment:
Thanks, I removed theses lines anyway as this wasn't the good approach, Twig extensions are able to inject globals without hacking DI as I did. 
(This didn't worked because of prepend priority: http://symfony.com/doc/current/bundles/prepend_extension.html#more-than-one-bundle-using-prependextensioninterface : The second global config preprend was skipped)